docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,74 @@
+[toc]
+
+# Run Antrea agent simulator
+
+## Table of Contents
+<!-- toc -->
+- [build the images](#build-the-images)
```

Comment:
toc should be capitalized, so do the section headers below.

Reply:
thanks, since the markdown is simple, I just removed the table of content